gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/FakeStorageRpc.java

```diff
@@ -38,6 +40,8 @@
 @NotThreadSafe
 public final class FakeStorageRpc implements StorageRpc {
 
+  // TODO(#829): Generation support.
+
   // fullname -> metadata
   private final Map<String, StorageObject> stuff = new HashMap<>();
```

Comment:
Unrelated to this PR: the variable name "stuff" could perhaps be more descriptive